.github/workflows/publish-package.yml

```diff
@@ -40,5 +34,36 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - uses: actions/checkout@v2
```

Comment:
line 15, better to use actions/checkout@v2 as well

Reply:
done